src/main/java/no/sikt/sws/WorkspaceIndexHandler.java

```diff
@@ -34,15 +34,19 @@ protected IndexResponse processInput(String body, RequestInfo request, Context c
         try {
             var url = workspace + "-" + index;
             logger.info("URL: " + url);
-            var response = openSearchClient.sendRequest(httpMethod, url, body);
+
+            var response = openSearchClient.sendRequest(
+                    httpMethod,
+                    url,
+                    RequestUtil.addWorkspace(body, workspace, index));
```

Comment:
Is this needed? Is there a usercase in which you put a new index with a config, and that config has the workspace in it?

Reply:
"workspaces" are a concept that we implement to hide other indexes from each client, this code translate between gateway and opensearch.

Reply:
However, I think this should be moved to the client, to ensure that all request uses the same informationflow.
If you have an idea about a more descriptive name, please enlighten me.